### PR TITLE
November Spotlights

### PR DESCRIPTION
**Pull request by Shadowthegod**
### Description
Article and the images for the spotlight of November. Announcing new members and Onosakihito's retire

### Status
Finished; Pending Review 

### Changelog
- Article and images are added
- Paragraph to announce new members added

### CODE REVIEW

File: news/2017-12-18-beatmap-spotlights-november-2017.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+* **osu!:** [Battle](https://osu.ppy.sh/u/4037545), [Halfslashed](https://osu.ppy.sh/u/4598899), [Delis](https://osu.ppy.sh/u/1603923), [Regou](https://osu.ppy.sh/u/419954), [Crystal](https://osu.ppy.sh/u/1646397), [Gamu](https://osu.ppy.sh/u/611174)
+* **osu!taiko:** [hikikochan](https://osu.ppy.sh/u/6512678), [Neppu](https://osu.ppy.sh/u/605973)
+* **osu!catch:** [Crystal](https://osu.ppy.sh/u/1646397)
+* **osu!mania:** [Garalulu](https://osu.ppy.sh/u/757783), [Pope Gadget](https://osu.ppy.sh/u/2288341), [Asherz007](https://osu.ppy.sh/u/9014047), [TheToaphster](https://osu.ppy.sh/u/7616811)
+
+# November 2017 Spotlights
+
+## osu!
+
+[![](/wiki/shared/news/2017-12-18-beatmap-spotlights-july-august-2017/shinkirou.jpg)](http://osu.ppy.sh/s/628765)
```
Comment (MegaApplePi) [inline news/2017-12-18-beatmap-spotlights-november-2017.md]: change `july-august` to `november`

Comment (Shadowthegod) [reply news/2017-12-18-beatmap-spotlights-november-2017.md]: I can't believe I forgot this. 😱

**Comment by Xinnoh** [comment]
There's a lot of inconsistencies with banners and who the set is by:

- Avalanche shows hybrid GDers and not set owner
- Nobore shows set owner, not hybrid GDer -> Should only show Eraser
- Dreadnaught + Hyakka Ryouran shows owner and not GDer -> Add GDer
- A Bella shows owner and GDers
- Arcadia + Gokuraku Jodo only shows owner, but there's 6 GDers on each so understandable
- Kantsubaki is a collab, but only shows owner -> Add fish39

**Comment by Shadowthegod** [comment]
Should be fixed now. I blame myself for unable to be able to pay attention and trying to heavily multitask between 6 windows. :psy:

**Pull request merged**